templates/azure/fips.html

```diff
@@ -0,0 +1,429 @@
+          <tr>
+            <td aria-label="Instance">DS1</td>
+            <td aria-label="Category">General Purpose</td>
+            <td aria-label="Cores" class="u-align--right">1</td>
+            <td aria-label="Ram" class="u-align--right">3.5GB</td>
+            <td aria-label="Azure Infrastructure" class="u-align--right">$0.077</td>
+            <td aria-label="Ubuntu Pro" class="u-align--right">$0.01</td>
+            <td aria-label="Total hourly" class="u-align--right">$0.087</td>
+          </tr>
+          <tr>
+            <td aria-label="Instance">DS2</td>
+            <td aria-label="Category">General Purpose</td>
+            <td aria-label="Cores" class="u-align--right">2</td>
+            <td aria-label="Ram" class="u-align--right">7GB</td>
+            <td aria-label="Azure Infrastructure" class="u-align--right">$0.154</td>
+            <td aria-label="Ubuntu Pro" class="u-align--right">$0.022</td>
+            <td aria-label="Total hourly" class="u-align--right">$0.176</td>
+          </tr>
+        </tbody>
+      </table>
+      <p>For complete pricing, see <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips?tab=Overview" class="p-link--external">Ubuntu Pro FIPS 18.04 LTS on the Azure Marketplace</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru is-dark">
+  <div class="row">
+    <div class="col-12">
+      <h2>Get Ubuntu Pro FIPS on Azure now:</h2>
+    </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips?tab=Overview" class="p-link--inverted">18.04 LTS</a></h3>
+    </div>
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial-fips?tab=Overview" class="p-link--inverted">16.04 LTS</a></h3>
+    </div>
+  </div>
+  <div class="row">
+    <p class="u-no-margin--bottom"><strong>More about Ubuntu Pro FIPS:</strong> <a href="https://assets.ubuntu.com/v1/3c96f340-Ubuntu_Pro_FIPS_Azure.pdf" class="p-link--inverted">Read the datasheet</a></p>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h2>Compliance and the most comprehensive open source security coverage - that’s Ubuntu Pro</h2>
+      <p>Ubuntu Pro FIPS images are based on Ubuntu Pro premium images, with security coverage included for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. This makes it ideal for companies that have embraced open source for their new products and need to ensure security for production and mission-critical workloads.</p>
+      <p><a href="https://ubuntu.com/azure/pro">Learn more about Ubuntu Pro on Azure ></a></p>
```

Comment:
We usually use this `&nbsp;&rsaquo;` instead of >

```suggestion
      <p><a href="https://ubuntu.com/azure/pro">Learn more about Ubuntu Pro on Azure&nbsp;&rsaquo;</a></p>
```

Reply:
Changed!